openstack/pipeline/02configure

```diff
@@ -28,6 +28,11 @@ else
     MOD_PARAMS[__NUM_NEUTRON_GATEWAY_UNITS__]=0
 fi
 
+# Default to OVN for > Train
+if has_min_release ussuri && ! has_opt --ovs; then
```

Comment:
--ml2-ovs, or an alias should be added below

Reply:
Good catch.